tests/fixtures.ts

```diff
@@ -121,6 +157,28 @@ export const test = baseTest.extend<TestFixtures, WorkerFixtures>({
 
   mcpBrowser: ['chrome', { option: true }],
 
+  mcpExtension: [false, { option: true }],
+
+  mcpExtensionPage: async ({ mcpExtension, mcpHeadless }, use) => {
+    if (!mcpExtension)
+      return await use(undefined);
+    const pathToExtension = path.join(url.fileURLToPath(import.meta.url), '../../extension');
+    const context = await chromium.launchPersistentContext('', {
+      headless: mcpHeadless,
+      args: [
+        `--disable-extensions-except=${pathToExtension}`,
+        `--load-extension=${pathToExtension}`,
+        '--enable-features=AllowContentInitiatedDataUrlNavigations'
```

Comment:
is this still required? if yes, do user browsers have it by default?

Reply:
Yes, navigating to data URLs is not allowed otherwise, needed for our tests.